src/app/content/highlights/hooks/locationChange.ts

```diff
@@ -25,6 +25,8 @@ const hookBody = ({dispatch, getState, highlightClient}: MiddlewareAPI & AppServ
   if (highlights.data) {
     dispatch(receiveHighlights(highlights.data));
   }
+
+  dispatch(addCurrentPageToSummaryFilters());
```

Comment:
you could just compose this logic here by calling a function, no need to dispatch an extra method

Reply:
I've made it separate to make tests easier to write, like:
```
it('initially has the selected page checked', () => {
    store.dispatch(receiveBook(book));
    hook(addCurrentPageToSummaryFilters());
```

Can I achive the same by calling `receiveUser` (`locationChange` is hooked to this)? I'll check